src/Tooltip/Tooltip.stories.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withKnobs, text, select } from '@storybook/addon-knobs'
+import { defaultTemplate } from '../../storybook/decorators/storyTemplates'
+import { Button, OverlayTrigger, Tooltip } from '../index.js'
+
+const stories = storiesOf('Tooltip', module)
+const description = (
+  <p>
+    This component is based on React Bootstrap Tooltip component. See{' '}
+    <a href="https://react-bootstrap.github.io/components.html#tooltips">
+      React Bootstrap Docs
+    </a>{' '}
+    for complete Tooltip component documentation.
+  </p>
+)
+stories.addDecorator(withKnobs)
+stories.addDecorator(
+  defaultTemplate({
+    title: 'Tooltip',
+    documentationLink:
+      'http://www.patternfly.org/pattern-library/widgets/#tooltip',
+    description: description
+  })
+)
+
+stories.addWithInfo('Tooltip', () => {
+  const tooltip = (
+    <Tooltip id="tooltip">
+      <strong>Holy guacamole!</strong> Check this info.
+    </Tooltip>
+  )
+  const placement = select(
+    'Placement',
+    ['top', 'bottom', 'left', 'right'],
+    'right'
+  )
+  const trigger = text('Trigger', 'hover focus')
```

Comment:
I would use a select knob here w/ options click, hover, and focus instead of a text knob.

Reply:
HI @priley86 , because addon Knobs doesn't support multiple select, so I have to use text with blank space.

Reply:
Hey @dabeng - my preference is to use single select for now (as users will likely not know what to type into this textbox). I understand that we can indicate both Hover + Focus triggers, but I think it's OK to only showcase one at a time for now. It looks like multiselect will be coming soon enough to Storybook Knobs, so we could convert to multi later is my thinking.
https://github.com/storybooks/storybook/issues/806

Reply:
Is it possible to use the single select menu to display all combinations of options? For example:
- hover
- focus
- hover focus

Reply:
👍  yes... thanks @jgiardino !